lib/teleterm/clusters/cluster_servers.go

```diff
@@ -68,3 +71,73 @@ func (c *Cluster) GetServers(ctx context.Context) ([]Server, error) {
 
 	return results, nil
 }
+
+func (c *Cluster) GetServers(ctx context.Context, r *api.GetServersRequest) (*GetServersResponse, error) {
+	var (
+		resp        *types.ListResourcesResponse
+		authClient  auth.ClientI
+		proxyClient *client.ProxyClient
+		err         error
+	)
+
+	err = addMetadataToRetryableError(ctx, func() error {
+		proxyClient, err = c.clusterClient.ConnectToProxy(ctx)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		defer proxyClient.Close()
+
+		authClient, err = proxyClient.ConnectToCluster(ctx, c.clusterClient.SiteName)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		defer authClient.Close()
+		sortBy := types.GetSortByFromString(r.SortBy)
+
+		resp, err = authClient.ListResources(ctx, proto.ListResourcesRequest{
+			Namespace:           defaults.Namespace,
+			ResourceType:        types.KindNode,
+			Limit:               r.Limit,
+			SortBy:              sortBy,
+			PredicateExpression: r.Query,
+			SearchKeywords:      client.ParseSearchKeywords(r.Search, ' '),
+			UseSearchAsRoles:    r.SearchAsRoles == "yes",
+		})
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+		return nil
+	})
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	clusterServers, err := types.ResourcesWithLabels(resp.Resources).AsServers()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	results := []Server{}
+	for _, server := range clusterServers {
+		results = append(results, Server{
+			URI:    c.URI.AppendServer(server.GetName()),
+			Server: server,
+		})
+	}
+
+	return &GetServersResponse{
+		Servers:    results,
+		StartKey:   resp.NextKey,
+		TotalCount: resp.TotalCount,
+	}, nil
+}
+
+type GetServersResponse struct {
+	// Resources is a list of resource.
```

Comment:
```suggestion
	// Servers is a list of servers.
```

Or just remove the comment, it's not adding much.

Reply:
Removing